### PR TITLE
Revamp feed configuration and implementation loading

### PR DESCRIPTION
**Pull request by tim1234ltp**
This PR revamped the configuration directory structure such that feed specfic config json and implementation python code are in `feeds/`
`config/config.json` will only contain generic metadata for starting the app

This should address #12

### CODE REVIEW

File: src/scraper/scrape.py [hunk 1]
```diff
@@ -30,6 +32,9 @@ def check_config(config: Dict):
     assert config["s3_bucket"]["public_key"]
     assert config["s3_bucket"]["secret_key"]
 
+    assert config["mobilitydatabase"]["url"]
+    assert config["mobilitydatabase"]["token"]
```
Comment (geeli123) [inline src/scraper/scrape.py:36]: are all configs going to be through mobility database?

File: src/scraper/scrape.py [hunk 2]
```diff
@@ -47,13 +52,33 @@ def create_s3_client(s3_config: Dict):
...
+    feed_class = getattr(module, feed_id.upper().replace("-", "_"))
+    feed = feed_class()
+    LOG.info("Start scraping", s3_bucket=s3_client)
+    while True:
+        content = feed.scrape()
+        now = datetime.now()
+        file_path = f'{feed_id}/{now.year}/{now.month}/{now.day}/{now.timestamp()}.binpb'
+        s3_file_path = f"raw/{file_path}"
+        os.makedirs(os.path.dirname(f"{DATA_DIR}/{file_path}"), exist_ok=True)
+        f = open(f"{DATA_DIR}/{file_path}", "wb")
```
Comment (geeli123) [inline src/scraper/scrape.py:66]: can use `with open(...) as f` to handle open and closing the file

Comment (geeli123) [reply src/scraper/scrape.py:66]: I also don't think you need to actually write this file to disk? Can directly upload the binary to s3
`s3. upload_fileobj`

File: src/scraper/scrape.py (hunk 1 above)
Comment (tim1234ltp) [reply src/scraper/scrape.py:36]: I have mandated that to be there for feeds.jsons falling back

File: src/scraper/scrape.py (hunk 2 above)
Comment (tim1234ltp) [reply src/scraper/scrape.py:66]: I was imagining a use case where there is issue with the files but scrapper is running. One can then shell into the docker and inspect the file on disk. And even manually upload them to ts3.
What's your thought?

Comment (geeli123) [reply src/scraper/scrape.py:66]: is there a reason not to just inspect in s3? I'd imagine it's the same right?

Primarily worried about long-term taking up too much disk space if we're not cleaning it up

**Review by geeli123 — APPROVED** [review]

**Pull request merged**